android/test-app-util.gradle

```diff
@@ -145,3 +145,41 @@ ext.getReactNativeVersionNumber = { baseDir ->
     def (major, minor, patch) = manifest["version"].findAll(/\d+/)
     return (major as int) * 10000 + (minor as int) * 100 + (patch as int)
 }
+
+ext.getSigningConfigs = { baseDir ->
```

Comment:
Can we add tests for this new function? See https://github.com/microsoft/react-native-test-app/blob/trunk/test/android-test-app/test-app-util.test.js

Reply:
Added a simple test :) Thanks!